lib/cucumber/core/test/case.rb

```diff
@@ -9,7 +9,7 @@ module Test
       class Case
         attr_reader :id, :name, :test_steps, :location, :tags, :language, :around_hooks
 
-        def initialize(id, name, test_steps, location, tags, language, around_hooks = [])
+        def initialize(id, name, test_steps, location, tags, language, around_hooks = []) # rubocop:disable Metrics/ParameterLists
```

Comment:
have these as items to fix in the todo, let the gem handle auto-generation. Then people can fix them up as/when

Reply:
So it's better to have cops disablements in `rubocop.yml`? I thought it was better to have them inline so that the cop is disabled on one line but active for the remaining of the file.

Reply:
You can have the todo exclude based on a file, yes it would cover the file but then it's a single place to fix it up. Also there will be a number of offenses to fix which helps Autofix / Manual fix when eliminating issues. The in-line isn't as useful for that.

Reply:
I have removed the inlined `rubocop:disable` and regenerated the `.rubocop_todo.yml` file as suggested.